lib/glimesh/chat.ex

```diff
@@ -388,4 +422,13 @@ defmodule Glimesh.Chat do
 
     {:ok, bad_user}
   end
+
+  defp broadcast_delete({:ok, channel_id, message_id}, :message_deleted) do
```

Comment:
Do we need `channel_id` here? Or can we broadcast the chat_message instead like the above `defp broadcast({:ok, chat_message}, event) do`